### DISCUSSION TITLE
Using mpv scripts with Jellyfin Media Player

### DISCUSSION THREAD

**Discussion by thx4nothing** [jellyfin/jellyfin-desktop#110] [Q&A]
Hello,

I want to use the [mpv_sponsorblock](https://github.com/po5/mpv_sponsorblock) script with Jellyfin Media Player, as it is based on mpv, but I cannot get it to work. I put it in the scripts folder in the JellyfinMediaPlayer user folder (the one with jellyfinmediaplayer.conf etc.), but it doesn't get loaded and it doesn't skip anything. I used it with the normal mpv and it works there. I also tried setting a custom path in the mpv.conf file, but it also doesn't get loaded.

Did anyone got it (or scripts in general) working?

**Comment by cmm** [comment]
mpv scripts used to work for me just great on pre-2.0 JMP versions when put under `~/.local/share/jellyfinmediaplayer/scripts/`.  but that does not really answer your question I'm afraid, because with 2.0 that doesn't work any more (or it does -- just hold off on upgrading JMP indefinitely, enjoy using unsafe and deprecated qt5 web component, no biggie).

with 2.0 state directory became `~/.local/share/jellyfin-desktop/`, with the _actual_ data directories being per-profile under `profiles/` inside that.  profile names are randomly-generated (making declarative configuration not possible any more BTW, but who cares, what with all the rest), but at least the structure of those profile-specific data directories appears similar to how `~/.local/share/jellyfinmediaplayer/` used to look pre-2.0.

so it looks like you just need to put mpv scripts under `~/.local/share/jellyfin-desktop/profiles/<some-vomit>/scripts/` now, right?  except of course that doesn't work, because why would it lol.

but wait!  there's Settings -> Client Settings -> Manual MPV Configuration now!  you can just put `script="<path-to-you-script>"` there, right?  nope, doesn't seem to do anything either.

hope that helps.
